### PR TITLE
Feature/cvsb 167

### PR DESCRIPTION
**Pull request by mgherceanu**

### CODE REVIEW

File: src/app/app.scss [hunk 1]
```diff
@@ -146,15 +142,12 @@
 ion-item-divider {
   .label-ios {
     margin: 0 8px 0 0;
+    font-size: 15px;
```
Comment (danieliordache) [inline src/app/app.scss]: use a scss variable

File: src/pages/atf/atf-details/atf-details.scss [hunk 2]
```diff
@@ -6,4 +6,23 @@ ion-app.ios page-atf-details {
...
+  .label-custom-style {
+    color: color($colors, dark);
+  }
+
+  .value-custom-style {
+    color: $list-ios-item-font-color;
+  }
+
+  .number-custom-style {
+    font-size: 15px;
```
Comment (danieliordache) [inline src/pages/atf/atf-details/atf-details.scss]: use scss variable

File: src/app/app.scss (hunk 1 above)
Comment (mgherceanu) [reply src/app/app.scss]: done

File: src/pages/atf/atf-details/atf-details.scss (hunk 2 above)
Comment (mgherceanu) [reply src/pages/atf/atf-details/atf-details.scss]: done

**Pull request merged**